src/components/PositionCard/V2.tsx

```diff
@@ -118,32 +118,32 @@ export default function V2PositionCard({ pair, border, stakedBalance }: Position
         {showMore && (
           <AutoColumn gap="sm">
             <FixedHeightRow>
-              <Text fontSize={16} fontWeight={500}>
+              <Text fontSize={16} fontWeight={535}>
```

Comment:
do we have a standard typography for fontSize 16 fontWeight 535? We use/redeclare it a lot here and the file below